dts/arm64/broadcom/bcm2711.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 honglin leng <a909204013@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm64/armv8-a.dtsi>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a72";
+			reg = <0>;
+		};
+	};
+
+	timer {
+		compatible = "arm,armv8-timer";
+		interrupt-parent = <&gic>;
+		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 14 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 11 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>,
+			     <GIC_PPI 10 IRQ_TYPE_LEVEL
+			      IRQ_DEFAULT_PRIORITY>;
+	};
+
+	soc {
+		sram0: memory@200000 {
+			device_type = "memory";
```

Comment:
no `compatible` here?

Reply:
Yes, SRAM does not require compatibility, and Zephyr does not need to perform any operations on SRAM. You can see that none of the SRAM devices under arm64 have a compatible property.

Reply:
well, this is simply wrong, especially when you are using that for `zephyr,sram`. In that case the SRAM region must at least have a `compatible = "mmio-sram";`